Cargo.toml

```diff
@@ -62,8 +62,8 @@ ctaphid-dispatch = { git = "https://github.com/trussed-dev/ctaphid-dispatch.git"
 apdu-dispatch = { git = "https://github.com/trussed-dev/apdu-dispatch.git", rev = "915fc237103fcecc29d0f0b73391f19abf6576de" }
 littlefs2 = { git = "https://github.com/trussed-dev/littlefs2.git", rev = "ebd27e49ca321089d01d8c9b169c4aeb58ceeeca" }
 serde-indexed = { git = "https://github.com/sosthene-nitrokey/serde-indexed.git", rev = "5005d23cb4ee8622e62188ea0f9466146f851f0d" }
-trussed = { git = "https://github.com/trussed-dev/trussed.git", rev = "cff2e663841b6a68d3a8ce12647d57b2b6fbc36c" }
+trussed = { git = "https://github.com/Nitrokey/trussed.git", rev = "cb7bd328549293077cd9d6c2e9e42df62180db96" }
```

Comment:
Please use the tag [v0.1.0-nitrokey.18](https://github.com/Nitrokey/trussed/releases/tag/v0.1.0-nitrokey.18).

Reply:
Done